src/http/proxy_server.rs

```diff
@@ -109,6 +109,16 @@ async fn handle(
         .set_server_software("rymfony/rust/fastcgi-client")
         .set_server_protocol(http_version);
 
+    for header in headers.iter() {
+        let (name, value) = header;
```

Comment:
For the record, inferred types here are the following:

`(name: &hyper::header::HeaderName, value: &hyper::header::HeaderValue)`

Not sure it helps 😅